pull-request.sh

```diff
@@ -254,7 +264,22 @@ main () {
                                 "${ASSIGNEES}" "${REVIEWERS}" "${TEAM_REVIEWERS}"
 
         fi
+    fi
+}
+
+# Run curl with default values
+curl_wrapper() {
+    printf "curl -fsSL -H 'AUTH...' %s\n" "$*" >&2
+    set +e
```

Comment:
Just want to verify with you - this setting is scoped to the function, so when it exists we return to `set -e`?

Reply:
no, this is scoped to the current shell, this depends on the fact that `curl_wrapper` is called in a subshell (either `$(curl_wrapper ...)` or `(curl_wrapper ..)`. Overall I think it's better to rely on the global `set -e` instead of setting and unsetting it.